examples/all-clusters-app/all-clusters-common/src/EnergyEvseManager.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#include <EnergyEvseManager.h>
+
+using namespace chip::app;
+using namespace chip::app::Clusters;
+using namespace chip::app::Clusters::EnergyEvse;
+
+CHIP_ERROR EnergyEvseManager::Init()
+{
+    GetInstance()->Instance::Init();
+    return CHIP_NO_ERROR;
+}
+
+void EnergyEvseManager::Shutdown()
+{
+    GetInstance()->Instance::Shutdown();
+}
+
+void MatterEnergyEvsePluginServerInitCallback() {}
```

Comment:
This is supposed to be in the cluster impl or `src/app/util/util.cpp`, not in the app.  This is an SDK hook, not public app API.

Reply:
Ah ok - I copied from bridged-actions-stub.cpp. 
Looks like I randomly picked the wrong example!

Reply:
Changed.

Reply:
@jamesharrow Yeah, the fact that some of our examples do things wrong is ... don't get me started.  :(